javaslang/src/test/java/javaslang/control/EitherRightProjectionTest.java

```diff
@@ -169,6 +184,14 @@ public void shouldConvertRightProjectionOfRightToJavaOptional() {
         assertThat(Either.<Integer, String> right("1").right().toJavaOptional()).isEqualTo(Optional.of("1"));
     }
 
+    // -- transform()
+
+    @Test
+    public void shouldTransform() {
+        String transformed = of(42).transform(v -> String.valueOf(v.get()));
```

Comment:
* please make local variables `final`, it's how @danieldietrich prefers them :)
* you could chose slightly simpler values, e.g. `of(1)`